assets/sass/components/_place-context-header.scss

```
@@ -1,19 +1,43 @@
 .place-context-header {
-  background-color:  govuk-colour("light-grey");
-  padding: govuk-spacing(3);
-  border-left: $govuk-border-width solid govuk-colour("blue");
-  margin-bottom: govuk-spacing(0);
+  @include govuk-responsive-padding(2, "top");
+  @include govuk-responsive-padding(2, "bottom");
+  background-color: govuk-colour("blue");  
```

When the banner is shown, we can still see the border from the GOVUK Phase banner. Could we move the place context up 1px in case it follows? Would be a _tiny bit_ neater :)

```suggestion
  background-color: govuk-colour("blue");  
  
  .govuk-phase-banner + & {
    margin-top: -1px;
  }
```